compair/api/assignment.py

```diff
@@ -381,10 +400,20 @@ def post(self, course_uuid):
         if new_assignment.compare_end is not None:
             new_assignment.compare_end = dateutil.parser.parse(params.get('compare_end', None))
 
+        new_assignment.enable_self_evaluation = params.get('enable_self_evaluation', False)
```

Comment:
`new_assignment.enable_self_evaluation = params.get('enable_self_evaluation', False)` is present here and on line 422 (please remove one)